resources/Lights/envmap_shader.mtlx

```diff
@@ -1,11 +1,17 @@
 <?xml version="1.0"?>
 <materialx version="1.37">
+
+   <!-- Custom node: <viewdir>                     -->
+   <!-- Required by environment shader graph below -->
+   <nodedef name="ND_viewdir_vector3" node="viewdir">
+     <output name="out" type="vector3" default="0.0, 0.0, 1.0"/>
+   </nodedef>
+  <implementation name="IM_viewdir_vector3_genglsl" nodedef="ND_viewdir_vector3" target="genglsl"/>
```

Comment:
This file is loaded explicitly by the viewer, so the <viewdir> node is only available here.